apps/settings/sub_menu/code_options_controller.cpp

```diff
@@ -93,6 +133,23 @@ void CodeOptionsController::willDisplayCellForIndex(HighlightCell * cell, int in
     mySwitch->setState(GlobalPreferences::sharedGlobalPreferences()->cursorSaving());
   }
 #endif
-}
 
+  if(index == 3)
```

Comment:
Can you indent it ?

Reply:
no I'll leave it like that it's not a big problem :p